package-lock.json

```diff
@@ -0,0 +1,10411 @@
+{
```

Comment:
Can you remove this file.? This module use yarn and it already have a yarn.lock file. Instead of npm install you can do yarn install.

Reply:
Done!